### PR TITLE
Bug 1240970 - Design/Copy/UX for Hello FTU Initiator Slideshow

### PR DESCRIPTION
**Pull request by chrafuse**
@dmose Slide show copy for both link generator and clicker string properties.

### CODE REVIEW

File: locale/en-US/standalone.properties [hunk 1]
```diff
@@ -25,6 +25,11 @@ vendorShortname=Mozilla
 
 call_progress_getting_media_description={{clientShortname}} requires access to your camera and microphone.
 
+## First Time Experience Slide
+intro_slide_title=You’re about to start browsing the Web with a friend
+intro_slide_copy=You’ll be able to see the page they shared with you and even chat over text or video — just don’t forget to let Hello use your camera and mic.
```
Comment (dmose) [inline locale/en-US/standalone.properties]: Hello wants to be {{clientSuperShortname}}

File: locale/en-US/add-on.properties [hunk 2]
```diff
@@ -37,6 +37,16 @@ first_time_experience_subheading2=Click the Hello button to browse Web pages wit
 first_time_experience_content=Use it to plan together, work together, laugh together.
 first_time_experience_button_label2=See how it works
 
+## First Time Experience Slides
+fte_slide_1_title=Browse Web pages with a friend
+fte_slide_1_copy=Whether you’re planning a trip or shopping for a gift, Firefox Hello lets you make faster decisions in real time.
+fte_slide_2_title=Get on the same page
+fte_slide_2_copy=Use the built-in text or video chat to share ideas, compare options and come to an agreement.
+fte_slide_3_title=Invite a friend by sending a link
+fte_slide_3_copy=Hello works with most desktop browsers. No accounts are necessary and everyone connects for free.
```
Comment (dmose) [inline locale/en-US/add-on.properties]: Hello wants to be {{clientSuperShortname}}

File: locale/en-US/add-on.properties [hunk 3]
```diff
@@ -37,6 +37,16 @@ first_time_experience_subheading2=Click the Hello button to browse Web pages wit
...
+fte_slide_4_title=Find the Hello icon to get started
```
Comment (dmose) [inline locale/en-US/add-on.properties]: Same here.

File: locale/en-US/add-on.properties [hunk 4]
```diff
@@ -37,6 +37,16 @@ first_time_experience_subheading2=Click the Hello button to browse Web pages wit
 first_time_experience_content=Use it to plan together, work together, laugh together.
 first_time_experience_button_label2=See how it works
 
+## First Time Experience Slides
+fte_slide_1_title=Browse Web pages with a friend
+fte_slide_1_copy=Whether you’re planning a trip or shopping for a gift, Firefox Hello lets you make faster decisions in real time.
```
Comment (dmose) [inline locale/en-US/add-on.properties]: Firefox Hello (and other instances of it) wants to be {{clientShortname2}}, I think.

File: locale/en-US/add-on.properties [hunk 5]
```diff
@@ -37,6 +37,16 @@ first_time_experience_subheading2=Click the Hello button to browse Web pages wit
...
+fte_slide_2_title=Get on the same page
+fte_slide_2_copy=Use the built-in text or video chat to share ideas, compare options and come to an agreement.
+fte_slide_3_title=Invite a friend by sending a link
+fte_slide_3_copy=Hello works with most desktop browsers. No accounts are necessary and everyone connects for free.
+fte_slide_4_title=Find the Hello icon to get started
+fte_slide_4_copy=Once you’ve found a page you want to discuss, click the icon in Firefox to create a link. Then send it to your friend however you like!
```
Comment (dmose) [inline locale/en-US/add-on.properties]: Firefox wants to be {{brandShortname}}

**Comment by chrafuse** [comment]
@dmose  Added variables and translation notes to strings.

**Comment by dmose** [comment]
Looks good; r=dmose

**Pull request merged**